docs/index.rst

```diff
@@ -1,24 +1,186 @@
-.. Confluent REST Proxy documentation master file, created by
-   sphinx-quickstart on Wed Dec 17 14:17:15 2014.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. _kafkarest_intro:
 
-Confluent REST Proxy
-====================
+Kafka REST Proxy
```

Comment:
I went through quite a lot of discussion with Marketing about this, and they came down pretty solidly on the "Confluent REST Proxy" side. I updated this topic in [DOCS-627: Rename Confluent Kafka REST Proxy to Confluent REST Proxy](https://github.com/confluentinc/kafka-rest/pull/453).

Reply:
@JimGalasyn okay great, i'll update. i'm also going to add a text-string replace for this, to avoid future missteps.

Reply:
https://github.com/confluentinc/docs/pull/1170